internal/services/costmanagement/export_resource_base.go

```diff
@@ -321,7 +341,18 @@ func expandExportDefinition(input []interface{}) *exports.ExportDefinition {
 	}
 
 	attrs := input[0].(map[string]interface{})
+	var dataset *exports.ExportDataset
+
+	if v, ok := attrs["data_version"]; ok {
+		dataset = &exports.ExportDataset{
+			Configuration: &exports.ExportDatasetConfiguration{
+				DataVersion: utils.String(v.(string)),
```

Comment:
The `utils` package is being deprecated, we should use `pointer.To` here
```suggestion
				DataVersion: pointer.To(v.(string)),
```